### PR TITLE
Bring codeowners file a bit more up to date

### PR DESCRIPTION
**Pull request by swissspidy**
Trying to reflect the new team structures.

Feedback welcome.

For reference: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

### CODE REVIEW

**Review by dvoytenko — COMMENTED** [review]
That changes themselves are LGTM. However, I'm wondering if this is a right time for us to drop auto-assignment? Having 8 people autoassigned to review is too many, imho. It's never clear anymore who is reviewing and how's punting. I think we can even allow folk to assign manually and only ensure that owner is required before merge?

**Comment by swissspidy** [comment]
Agreed. We had some discussion around this which I need to follow-up.

I was hoping to use team aliases for this, which would make the pod involvement clearer and not auto-assign half a dozen people.

Will look into this again, considering this a WIP for now.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,39 +1,72 @@
...
+#
+# Team Pods
+#
+# Workspace: @dvoytenko @wassgha @barklund @miina @obetomuniz @ndev1991 @merapi
+#
+# WordPress & Infrastructure: @swissspidy @spacedmonkey
+#
+# Dashboard & Templates: @carlos-kelly @BrittanyIRL @mariano-formidable @littlemilkstudio
+#
+# NOTE: Most owners are currently commented out since we cannot currently use GitHub team aliases.
```
Comment (swissspidy) [inline .github/CODEOWNERS:12]: The goal is less noise due to auto review requests

**Review by spacedmonkey — APPROVED** [review]
Looks good to me

**Pull request merged**